### PR TITLE
Fix: LLM API logits processor example comments

### PR DESCRIPTION
**Pull request by syuoni**
This MR fixes LLM API logits processor example comments.

### CODE REVIEW

**Comment by niukuo** [comment]
[/LLM/main/L0_MergeRequest_PR pipeline #182](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/182/) completed with status: 'FAILURE'

**Comment by niukuo** [comment]
[/LLM/main/L0_MergeRequest_PR pipeline #185](https://nv/trt-llm-cicd/job/main/job/L0_MergeRequest_PR/185/) completed with status: 'FAILURE'

**Comment by niukuo** [comment]
[PR_Github #191](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/191/) [ run ] triggered by Bot

**Comment by niukuo** [comment]
[PR_Github #191](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/191/) [ run ] completed with state `FAILURE`

**Review by kaiyux — APPROVED** [review]

**Comment by kaiyux** [comment]
@syuoni The DCO checking is failed - please sign off the commit by adding `-s` when commit, so the command becomes  `git commit -s -m "Msg"`

**Comment by niukuo** [comment]
[PR_Github #204](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/204/) [ run ] triggered by Bot

**Comment by kaiyux** [comment]
> @syuoni The DCO checking is failed - please sign off the commit by adding `-s` when commit, so the command becomes `git commit -s -m "Msg"`

Never mind, I rebased your branch and the DCO check seems to be passing.

**Comment by niukuo** [comment]
[PR_Github #207](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/207/) [ run ] triggered by Bot

**Comment by niukuo** [comment]
[PR_Github #204](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/204/) [ run ] completed with state `ABORTED`

**Comment by kaiyux** [comment]
/bot skip --comment "style check passed on comment change"

**Comment by niukuo** [comment]
[PR_Github #215](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/215/) [ skip ] triggered by Bot

**Comment by niukuo** [comment]
[PR_Github #207](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/207/) [ run ] completed with state `ABORTED`

**Comment by niukuo** [comment]
[PR_Github #215](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/215/) [ skip ] completed with state `SUCCESS`

**Comment by ZhanruiSunCh** [comment]
/bot skip --comment "style check passed on comment change"

**Comment by niukuo** [comment]
[PR_Github #226](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/226/) [ skip ] triggered by Bot

**Comment by niukuo** [comment]
[PR_Github #226](https://nv/trt-llm-cicd/job/helpers/job/PR_Github/226/) [ skip ] completed with state `SUCCESS`
Skipping testing for commit `7be8e60`
